src/module/interface/user/index.ts

```diff
@@ -11,6 +11,7 @@ export interface IUser {
   name: string;
   title?: string;
   job?: Job;
+  lastUpdate?: Record<string, string>;
```

Comment:
The new metadata property on user objects indicating the last update viewed for each timeline.